src/helpers/web3.ts

```diff
@@ -1,6 +1,8 @@
 import assert from 'assert'
 import Web3 from 'web3'
-import { HttpProvider, WebsocketProvider } from 'web3-core'
+// TODO: check if latest version fixes
```

Comment:
@Velenir can you add a recurring calendar event and maybe an issue to do this check so we don't forget